pages/index.js

```diff
@@ -11,99 +14,125 @@ import IndexIsssueListItem from "../components/shared/IndexIssueListItem";
-            </h2>
-          </Col>
+        <Row className="py-5 text-primary text-center">
+          <h2>
+            <strong>We&apos;re here to help</strong>
+          </h2>
         </Row>
-        <Row className="text-primary bg-light">
-          <Col md={2}></Col>
+        <Row className="text-primary text-center">
+          <Col sm={1}></Col>
           <ServiceItem
             content="Modernize software development processes while introducing good agile and human-centered design practices"
-            icon={<FaUsersCog style={{ verticalAlign: "top" }} />}
+            icon={
+              <IoIosPeople style={{ verticalAlign: "top", fontSize: "3rem" }} />
+            }
           />
-          <Col md={2}></Col>
           <ServiceItem
-            content="Digitize and streamline internal systems to save time and unlock insights from data"
-            icon={<FaBusinessTime style={{ verticalAlign: "top" }} />}
+            content="Improve public-facing services like websites or applications"
+            icon={<MdWeb style={{ verticalAlign: "top", fontSize: "3rem" }} />}
           />
-          <Col md={2}></Col>
-        </Row>
-        <Row className="pb-4 text-primary bg-light">
-          <Col md={2}></Col>
           <ServiceItem
-            content="Improve public-facing services like websites or applications"
-            icon={<MdWeb style={{ verticalAlign: "top" }} />}
+            content="Digitize and streamline internal systems to save time and unlock insights from data"
+            icon={<FaBriefcase style={{ verticalAlign: "top" }} />}
           />
-          <Col md={2}></Col>
           <ServiceItem
             content="Manage geospatial data and publish it on the web"
             icon={<FaGlobe style={{ verticalAlign: "top" }} />}
           />
-          <Col md={2}></Col>
+          <Col sm={1}></Col>
         </Row>
+        <hr className="my-3 mx-4"></hr>
         <Row className="pt-3 pb-4 text-primary">
-          <Col md={2}></Col>
+          <Col sm={1}></Col>
           <Col>
-            <h2>
-              <strong>Featured projects</strong>
-            </h2>
+            <h4 className="fw-bold">Featured</h4>
           </Col>
         </Row>
         <Row>
-          {projects.map((project) => {
-            return project.isFeatured ? (
-              <Col key={project.number} md={3} className="m-0 p-2">
-                <IndexIsssueListItem type="project" issue={project} />
-              </Col>
-            ) : null;
-          })}
+          <Col>
+            <Carousel
+              variant="dark"
+              indicators={false}
+              nextIcon={<IoIosArrowDropright className="fs-1 text-primary" />}
+              prevIcon={<IoIosArrowDropleft className="fs-1 text-primary" />}
+              interval={null}
+            >
+              <Carousel.Item>
+                <Row>
+                  <Col sm={1}></Col>
+                  {featuredIssues.slice(0, 4).map((issue) => {
```

Comment:
This doesn't feel super elegant and the code is a little redundant. It also requires us to only have 8 featured issues so its not very reusable if we want more or less featured issues in the future. But since we want 4 cards on each "slide" of the carousel and 8 total cards, this was the solution I came up with. Let me know if you have other ideas!

Reply:
This is a good point, I'll definitely use your idea to refactor the carousel into it's own component. Thanks Mike!

Reply:
Also yeah, the way the cards are rendered in the carousel is the same way they are rendered in the [Projects](https://deploy-preview-50--atd-product.netlify.app/projects?status=in_progress) and [Products](https://deploy-preview-50--atd-product.netlify.app/products) pages, using the same `IndexIsssueListItem` component. So if I change the design for the carousel cards it will also change the cards on the other pages. @patrickm02L would you rather keep the design as is or change the image to being on the top half of the card which would also change how they look in those other pages?

Reply:
Please change the design of the cards so they mirror what's rendered in the Data & Performance hub. 
https://data.mobility.austin.gov/

Image
Title
Description

Reply:
what should happen for the cards that don't have images?

Reply:
@patrickm02L Just to double check, should this change also happen for the cards on the Products and Projects pages?

Reply:
@roseeichelmann yes ultimately all the cards will have the new design.